game.py

```diff
@@ -0,0 +1,19 @@
+class helping:
+  def menu():
+      print ("---------------------------------------------------------------")
+      print ('Lemonade Stand: cost $500 but you add $100 to your revenue, to buy answer ls.\n')
```

Comment:
Great work! I do think Lemonade Stands should give $120 revenue.. what do you think?

Reply:
tat Is not the point I made a whole algorithm on how much stuff should cost starting with the lemonade stand